### ISSUE TITLE
make build-kfctl is failed during test

### ISSUE THREAD

**Issue by gabrielwen** [kubeflow/kubeflow#2715]
Seems like something wrong with deepcopy.  FYI, `make build-kfctl` works fine in local machine.  However, it fails on both JUNIT and Argo.

```
util.py                     71 INFO     Makefile:72: recipe for target 'pkg/apis/apps/gcp/v1alpha1/zz_generated.deepcopy.go' failed
util.py                     71 INFO     make: *** [pkg/apis/apps/gcp/v1alpha1/zz_generated.deepcopy.go] Error 255
util.py                     45 INFO     Running: make build-kfctl
cwd=/mnt/test-data-volume/kubeflow-presubmit-kfctl-go-2713-66ab02d-6337-0d08/src/kubeflow/kubeflow/bootstrap
util.py                     60 INFO     Subprocess output:
util.py                     71 INFO     /root/go/bin/deepcopy-gen -i github.com/kubeflow/kubeflow/bootstrap/pkg/apis/apps/gcp/... -O zz_g
enerated.deepcopy
util.py                     71 INFO     F0315 20:26:27.057077    8649 deepcopy.go:126] Failed loading boilerplate: open k8s.io/code-gener
ator/hack/boilerplate.go.txt: no such file or directory
util.py                     71 INFO     Makefile:72: recipe for target 'pkg/apis/apps/gcp/v1alpha1/zz_generated.deepcopy.go' failed
util.py                     71 INFO     make: *** [pkg/apis/apps/gcp/v1alpha1/zz_generated.deepcopy.go] Error 255
util.py                     45 INFO     Running: make build-kfctl
cwd=/mnt/test-data-volume/kubeflow-presubmit-kfctl-go-2713-66ab02d-6337-0d08/src/kubeflow/kubeflow/bootstrap
util.py                     60 INFO     Subprocess output:
util.py                     71 INFO     /root/go/bin/deepcopy-gen -i github.com/kubeflow/kubeflow/bootstrap/pkg/apis/apps/gcp/... -O zz_g
enerated.deepcopy
util.py                     71 INFO     F0315 20:26:37.651054    9499 deepcopy.go:126] Failed loading boilerplate: open k8s.io/code-gener
ator/hack/boilerplate.go.txt: no such file or directory
util.py                     71 INFO     Makefile:72: recipe for target 'pkg/apis/apps/gcp/v1alpha1/zz_generated.deepcopy.go' failed
util.py                     71 INFO     make: *** [pkg/apis/apps/gcp/v1alpha1/zz_generated.deepcopy.go] Error 255
util.py                     45 INFO     Running: make build-kfctl
cwd=/mnt/test-data-volume/kubeflow-presubmit-kfctl-go-2713-66ab02d-6337-0d08/src/kubeflow/kubeflow/bootstrap
util.py                     60 INFO     Subprocess output:
util.py                     71 INFO     /root/go/bin/deepcopy-gen -i github.com/kubeflow/kubeflow/bootstrap/pkg/apis/apps/gcp/... -O zz_g
enerated.deepcopy
util.py                     71 INFO     F0315 20:26:48.012311   10346 deepcopy.go:126] Failed loading boilerplate: open k8s.io/code-gener
ator/hack/boilerplate.go.txt: no such file or directory
util.py                     71 INFO     Makefile:72: recipe for target 'pkg/apis/apps/gcp/v1alpha1/zz_generated.deepcopy.go' failed
util.py                     71 INFO     make: *** [pkg/apis/apps/gcp/v1alpha1/zz_generated.deepcopy.go] Error 255
generated xml file: /mnt/test-data-volume/kubeflow-presubmit-kfctl-go-2713-66ab02d-6337-0d08/output/artifacts/junit_kfctl-build-test.xml
========================== 1 failed in 143.72 seconds ==========================
```

**Comment by gabrielwen** [comment]
confirmed it's because `GOPATH` is not set.

**Comment by lluunn** [comment]
is it fixed?

**Comment by gabrielwen** [comment]
the fix is in #2713

**Comment by gabrielwen** [comment]
#2713 might take longer, adding another PR to fix this #2729
